integrationtests/Paket.IntegrationTests/InstallSpecs.fs

```diff
@@ -541,7 +541,9 @@ let ``#1815 duplicate fsharp core reference when using netstandard1.6``() =
       |> Seq.toList
       |> Paket.LoadingScripts.PackageAndAssemblyResolution.getPackageOrderResolvedPackage
       |> Seq.collect (fun p ->
-        let installModel = paketDependencies.GetInstalledPackageModel(group, p.Name.ToString())
+        let installModel =
+          paketDependencies.GetInstalledPackageModel(group, p.Name.ToString())
+            .ApplyFrameworkRestrictions(Requirements.getRestrictionList p.Settings.FrameworkRestrictions)
```

Comment:
@forki Is this the correct thing to do, I have no idea. Should the public API already add the restrictions here?

Anyway thanks for bringing me on the right track!

Are there other things caller need to do?